lms/djangoapps/course_api/tests/test_views.py

```diff
@@ -419,7 +419,7 @@ def test_too_many_courses(self):
         self.setup_user(self.audit_user)
 
         # These query counts were found empirically
-        query_counts = [50, 46, 46, 46, 46, 46, 46, 46, 46, 46, 16]
+        query_counts = [51, 47, 47, 47, 47, 47, 47, 47, 47, 47, 17]
```

Comment:
Why did these numbers go up?

Reply:
Extra request was due to course_overviews.exclude in course overview `get_all_courses` method. Turned back.